packages/runtime/src/prop-controllers/instances.ts

```diff
@@ -81,12 +96,11 @@ class RichTextPropController extends PropController<RichTextPropControllerMessag
           case BuilderEditMode.BUILD:
             this.editor?.deselect().blur()
             break
-
-          case BuilderEditMode.CONTENT:
-            this.editor?.focus().moveToRangeOfDocument()
-            break
         }
-
+        break
+      }
+      case RichTextPropControllerMessageType.FOCUS: {
+        this.editor?.focus().moveToRangeOfDocument()
         break
```

Comment:
Open to pushback on this change. 

I could also update the `BuilderEditMode.CONTENT` to pass a `elementkey` + `propname`. 

That felt like the implementation creeping across the API, so I did this instead.

Reply:
Is this behavior different than the current rich text prop controller? If so, why does the behavior need to be different?

Reply:
Ok, so as I've reviewed the code I'm seeing that you've changed this across both the rich text control and the prop controller. Do you mind explaining why you introduced the change? I see you mentioned `elementKey` and `propName`, but I'm not seeing those used here so I don't understand the connection yet.

Reply:
When you have multiple panels, and you double click to enter content mode on the first rich text controller, then focus flickers through all rich text instances and lands on the last one. 

This is because we hadn't had this "multiple richtext panels visible at once" edge case before. This change handles that edge case.

Reply:
Got it. Are there any edge cases that were covered by the focus even that aren't covered by the builder mode switching? There's some very small nuances in this UX so I want to make sure that nothing breaks

Reply:
Not that I have found. I'll keep my eyes pealed!